spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/main/java/org/springframework/boot/gradle/tasks/bundling/LaunchScriptConfiguration.java

```diff
@@ -106,17 +106,12 @@ public boolean equals(Object obj) {
 			return false;
 		}
 		if (this.script == null) {
-			if (other.script != null) {
-				return false;
-			}
+			return other.script == null;
 		}
 		else if (!this.script.equals(other.script)) {
 			return false;
 		}
-		else if (!equalContents(this.script, other.script)) {
-			return false;
-		}
-		return true;
+		else return equalContents(this.script, other.script);
```

Comment:
@SaberMaycry Can you fix the checkstyle error on this line please?

Reply:
```suggestion
		else return equalContents(this.script, other.script);
```
Of course.